src/app/clusters/refrigerator-alarm-server/refrigerator-alarm-server.cpp

```diff
@@ -69,6 +69,21 @@ EmberAfStatus RefrigeratorAlarmServer::GetStateValue(EndpointId endpoint, BitMas
     return status;
 }
 
+EmberAfStatus RefrigeratorAlarmServer::GetSupportedValue(EndpointId endpoint, BitMask<AlarmMap> * supported)
+{
+    EmberAfStatus status = Attributes::Supported::Get(endpoint, supported);
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    {
+        printf("################################################################\n");
```

Comment:
why this extra noisiy printf? and why printf?

since the conditional is just for the logs, how about 

```
if (status != EMBER_ZCL_STATUS_SUCCES) 
{
   ChipLogProgress(...);
} else {
     ChipLogProgress(...);
}
return status;
```

Reply:
Removed printf and applied suggested changes.